File: SQLite_Example/app/src/main/java/com/example/sqlite/juan/activities/InsertActivity.java
package com.example.sqlite.juan.activities;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import com.example.sqlite.juan.R;
import com.example.sqlite.juan.helpers.Data;
import com.example.sqlite.juan.pojos.Usuario;

public class InsertActivity extends AppCompatActivity {

    EditText id,nombre,edad,correo;
    Button insert;
    Data data;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_insert);

        id =(EditText) findViewById(R.id.id);
        nombre =(EditText) findViewById(R.id.nombre);
        edad =(EditText) findViewById(R.id.edad);
        correo =(EditText) findViewById(R.id.correo);
        insert =(Button) findViewById(R.id.insert);

        insert.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Usuario user = new Usuario(id.getText().toString(),nombre.getText().toString(),
                        Integer.valueOf(edad.getText().toString()),correo.getText().toString());

                data = new Data(getApplicationContext());
                data.insertUsuario(user);
                Toast.makeText(getApplicationContext(),
                        "Se agrego el Usuario " + id.getText().toString(),
                        Toast.LENGTH_LONG).show();
                data.close();
            }
        });
    }
}

File: SQLite_Example/app/src/main/java/com/example/sqlite/juan/activities/InsertActivity.kt
package com.example.sqlite.juan.activities

import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.example.sqlite.juan.R
import com.example.sqlite.juan.helpers.Data
import com.example.sqlite.juan.pojos.Usuario

class InsertActivity : AppCompatActivity() {
    var id: EditText? = null
    var nombre: EditText? = null
    var edad: EditText? = null
    var correo: EditText? = null
    var insert: Button? = null
    var data: Data? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_insert)

        id = findViewById<View>(R.id.id) as EditText
        nombre = findViewById<View>(R.id.nombre) as EditText
        edad = findViewById<View>(R.id.edad) as EditText
        correo = findViewById<View>(R.id.correo) as EditText
        insert = findViewById<View>(R.id.insert) as Button

        insert!!.setOnClickListener {
            val user = Usuario(
                id!!.text.toString(),
                nombre!!.text.toString(),
                Integer.valueOf(edad!!.text.toString()),
                correo!!.text.toString()
            )
            data = Data(applicationContext)
            data!!.insertUsuario(user)
            Toast.makeText(
                applicationContext,
                "Se agrego el Usuario " + id!!.text.toString(),
                Toast.LENGTH_LONG
            ).show()
            data!!.close()
        }
    }
}